test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/NuGet.XPlat.FuncTest.csproj

```diff
@@ -35,7 +34,7 @@
   </ItemGroup>
   <Import Project="$(BuildCommonDirectory)common.targets" />
   <PropertyGroup>
-    <PostBuildEvent>xcopy /y $(MSBuildProjectDirectory)\bin\$(Configuration)\$(OutputPath)$(TargetFramework)\$(RuntimeIdentifier)\Microsoft.CSharp.Core.targets $(MSBuildProjectDirectory)\bin\$(Configuration)\$(OutputPath)$(TargetFramework)\$(RuntimeIdentifier)\Roslyn\</PostBuildEvent>
+    <PostBuildEvent Condition=" '$(IsXPlat)' != 'true' ">xcopy /y $(MSBuildProjectDirectory)\bin\$(Configuration)\$(OutputPath)$(TargetFramework)\$(RuntimeIdentifier)\Microsoft.CSharp.Core.targets $(MSBuildProjectDirectory)\bin\$(Configuration)\$(OutputPath)$(TargetFramework)\$(RuntimeIdentifier)\Roslyn\</PostBuildEvent>
```

Comment:
use the copy task instead to make it cross plat : https://msdn.microsoft.com/en-us/library/3e54c37h.aspx